File: web_vue.client/types/viewmodel/Auth/LoginFailViewModel.ts
// !! 此檔案由 generate-models.mjs 自動生成，請勿手動修改 !!
// 來源：Web_Vue.Server/ViewModels
import type { LoginFailTypeValue } from '../enum'

/** 登入失敗的附加資訊（放在 ResponseViewModel.Data） */
export interface LoginFailViewModel {
  /** 失敗類型 */
  FailType: LoginFailTypeValue
  /** 建議導向路徑（例如密碼過期時導向 /forget-password） */
  RedirectPath: string | null
}

File: Web_Vue.Server/ViewModels/Auth/LoginFailViewModel.cs
namespace Web_Vue.Server.ViewModels.Auth;

/// <summary> 登入失敗的附加資訊（放在 ResponseViewModel.Data） </summary>
public class LoginFailViewModel
{
    /// <summary> 失敗類型 </summary>
    public LoginFailType FailType { get; set; }

    /// <summary> 建議導向路徑（例如密碼過期時導向 /forget-password） </summary>
    public string? RedirectPath { get; set; }
}
